hash/x68k_flop.xml

```diff
@@ -8282,12 +8446,39 @@ Including "Daikairei Powerup kit to Shin Scenario Make kit" &amp; "Daikairei Tsu
 		</part>
 	</software>
 
-	<software name="laplace">
+	<!-- The user disk creation process doesn't recognize the data disk. This happens in both the original and cracked versions. -->
```

Comment:
`notes`, repeated twice in this and `laplacec` entry below (i.e. without the "This happens" extra phrase).

Reply:
Also done.